values.yaml

```diff
@@ -163,3 +163,7 @@ config:
 #  secretKey: define_your_secret
   disableInstaller: false
   offlineMode: false
+  require_signin: false
```

Comment:
Please use lowerCamelCase for all parameters

Reply:
Thank you for the PR. - LGTM, but please address @gs11 's comments.  I will merge it afterwards.

Reply:
Done!